File: Linguagem C++/Vetores/AbaixoDaMedia.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{
    int n;
    double soma,media;

    cout << "Quantos elementos vai ter o vetor? ";
    cin >> n;

    double vet[n];

    // Lendo o vetor
    for (int i=0;i<=n-1;i++){
        cout << "Digite um numero: ";
        cin >> vet[i];
    };

    cout << endl;
    //Fazendo a soma e a media do vetor
    for (int i=0;i<=n-1;i++){
        soma = soma + vet[i];
    };
    media = soma / n;
    cout << fixed << setprecision(3);
    cout << "MEDIA DO VETOR = " << media << endl;

    //Imprimindo os elemento abaixo da média
    cout << "ELEMENTOS ABAIXO DA MEDIA:" << endl;
    for (int i=0;i<=n-1;i++){
        if (vet[i] < media){
            cout << fixed << setprecision(1);
            cout << vet[i] << endl;
        };
    };

    return 0;
}

File: Linguagem Python/Vetores/AbaixoDaMedia.py
n = int(input("Quantos elementos vai ter o vetor? "))

vet: [int] = [0 for x in range(n)]

for i in range(0,n):
    vet[i] = int(input("Digite um numero: "))

cont = 0
soma = 0
for i in range(0,n):
    if vet[i] % 2 == 0:
        cont = cont + 1
        soma = soma + vet[i]

if cont == 0 :
    print("NENHUM NUMERO PAR")
else:
    media = soma / cont
    print(f"MEDIA DOS PARES = {media:.1f}")